### PR TITLE
add confirm step to some UI actions

### PR DESCRIPTION
**Pull request by timoschilling**
Actual a click on "revert" (stock firmware), "reset" (devise settings / hap settings) and "reboot" will be directly executed. I think that's not a good UI, unrevertible action should be confirmed by the user. I added the confirmations in this PR.

### CODE REVIEW

**Review by andyblac — COMMENTED** [review]
Should this not be asking "Are you sure you want this device to be reverted to stock firmware?", this sounds better English IMO.

**Comment by timoschilling** [comment]
I'm ok with any text.

**Comment by timoschilling** [comment]
I'm ok with any text, would you like to provide a text for each case?

**Comment by andyblac** [comment]
Maybe @rojer has a preference?

File: fs_src/script.js [hunk 1]
```diff
@@ -340,12 +342,16 @@ function gdoSetConfig(c, cfg, spinner) {
...
 el("reboot_btn").onclick = function () {
+  if(!confirm("Should this device perform a reboot?")) return;
+
   sendMessageWebSocket("Sys.Reboot", {delay_ms: 500}).then(function () {
     alert("System is rebooting and will reconnect when ready.");
   });
 }
 
 el("reset_btn").onclick = function () {
+  if(!confirm("Should this device perform a reset?")) return;
```
Comment (rojer) [inline fs_src/script.js:353]: "Device configuration will be wiped and return to AP mode. Are you sure?"

File: fs_src/script.js [hunk 2]
```diff
@@ -1040,6 +1046,8 @@ el("update_btn").onclick = function () {
   checkUpdate();
 };
 el("revert_btn").onclick = function () {
+  if(!confirm("Should this device perform a firmware reset?")) return;
```
Comment (rojer) [inline fs_src/script.js:1049]: "Revert to stock firmware?"

**Pull request merged**